stdlib/opal-platform.rb

```diff
@@ -3,13 +3,11 @@
 nashorn         = `typeof(Java) !== "undefined" && Java.type`
 headless_chrome = `typeof(navigator) !== "undefined" && /\bHeadlessChrome\//.test(navigator.userAgent)`
 
-case
-when nashorn
-  OPAL_PLATFORM = 'nashorn'
-when node
-  OPAL_PLATFORM = 'nodejs'
-when headless_chrome
-  OPAL_PLATFORM = 'headless-chrome'
-else # possibly browser, which is the primary target
-  OPAL_PLATFORM = nil
-end
+OPAL_PLATFORM = if nashorn
+                  'nashorn'
+                elsif node
+                  'nodejs'
+                elsif headless_chrome
+                  'headless-chrome'
+                else # possibly browser, which is the primary target
+                end
```

Comment:
still prefer the earlier version with its regular repetition, also the missing nil bothers me